SharedSource/Development/UnitTesting/Serializer.cs

```diff
@@ -27,7 +27,7 @@ namespace Remotion.Development.UnitTesting
   /// Provides quick serialization and deserialization functionality for unit tests.
   /// </summary>
   /// <remarks>The methods of this class use a <see cref="BinaryFormatter"/> for serialization.</remarks>
-  public static partial class Serializer
+  static partial class Serializer
```

Comment:
Good fix. Should be a dedicated commit since it's unrelated to the current task.